canonical/ubuntu.com
# bug: Change the regex that finds the CVE indentifier to ignore links that end in '.html'

## Done

- There is some regex we use to parse links in the cve notices description to make them link internally. This was also capturing a link that we wanted to be directed externally, this PR addresses that

## QA

- Go to https://ubuntu-com-14198.demos.haus/security/notices/USN-5142-1
- At the bottom of the 'Details' section there is an external link to Samba, in the above example https://www.samba.org/samba/security/CVE-2020-25717.html
- Check that the link works
- Check that the other links in 'Details' work

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14294
